getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+    "    noteable_features: str = Field(..., description=\"Any noteworthy features/achievements about the person\")\n",
+    "    hobbies: Optional[str] = Field(description=\"Anything hobbies the person may have\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 76,
+   "id": "c5d9b7cc-d127-4f28-ab8f-829f236ef419",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Person(name='Kate Sheppard', alive=False, profession='Suffragist', noteable_features=\"Kate Sheppard was the most prominent member of the women's suffrage movement in New Zealand. She played a key role in making New Zealand the first country to grant women the right to vote in 1893.\", hobbies=None)"
+      ]
+     },
+     "execution_count": 76,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "structured_model = model.with_structured_output(Person)\n",
+    "response = structured_model.invoke(\"Tell me about Kate Sheppard\")\n",
+    "response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52eb76bf-283c-4095-aa33-27e772f09292",
+   "metadata": {},
+   "source": [
+    "##### As the response of the invocation has been structured using the Person tool, the response can be accessed like a `Person` object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "id": "63f05dc8-7832-429c-b710-ceb72a56b492",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Kate Sheppard\n",
+      "False\n",
+      "Suffragist\n",
+      "Kate Sheppard was the most prominent member of the women's suffrage movement in New Zealand. She played a key role in making New Zealand the first country to grant women the right to vote in 1893.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(response.name)\n",
+    "print(response.alive)\n",
+    "print(response.profession)\n",
+    "print(response.noteable_features)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d5247e4-c810-4a61-98ee-9534ae10d300",
+   "metadata": {},
+   "source": [
+    "#### JSON"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 80,
+   "id": "2f688856-6992-4ed5-bf4b-62ddc592aee3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "350506c9-8d59-43c5-a5ba-cc2493da2faf",
+   "metadata": {},
+   "source": [
+    "Models can also be explicitly told to respond in a JSON structured format. This could then be used for future API calls or for easier access to information. However, **the word \"json\" must be included in the message string.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7aea759b-de79-48be-8ebe-c979f1c45bc3",
+   "metadata": {},
+   "source": [
+    "**latest update for LangChain seems to have broken the JSON output**"
```

Comment:
Maybe just remove this section for now if you don't think it will be fixed soon.

Reply:
One of the later updates seems to have corrected the issue, so I've reincluded the section again.